### ISSUE TITLE
Signing subkey expired in ansible-galaxy release

### ISSUE THREAD

**Issue by mixmasala** [systemli/ansible-role-hidden-service#25]
It looks like the baked-in torproject.asc contains an expired key:

```
gpg --edit-key A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89
gpg (GnuPG) 2.2.10; Copyright (C) 2018 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.


pub  rsa2048/EE8CBC9E886DDD89
     created: 2009-09-04  expires: 2020-08-29  usage: SC  
     trust: unknown       validity: unknown
sub  rsa2048/74A941BA219EC810
     created: 2009-09-04  expired: 2018-08-30  usage: S   
[ unknown] (1). deb.torproject.org archive signing key
```

**Comment by mixmasala** [comment]
It looks like this issue was fixed in git master but hasn't been updated on ansible-galaxy yet.

```
ansible-galaxy install systemli.hidden-service
- downloading role 'hidden-service', owned by systemli
- downloading role from https://github.com/systemli/ansible-role-hidden-service/archive/1.0.0.tar.gz
```

**Comment by t2d** [comment]
Thank you for raising the issue. I'm unsure, why galaxy didn't pick up the changes by itself. Maybe because it was only a merge request?! Anyway, I uploaded a new fixed version `1.1.1`.
